database/src/migrations/20230823000000_survey_partnerships.ts

```diff
@@ -0,0 +1,231 @@
+import { Knex } from 'knex';
+
+/**
+ * Adds new survey partnerships tables
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+
+  ----------------------------------------------------------------------------------------
+  -- Remove old view
+  ----------------------------------------------------------------------------------------
+
+  SET search_path=biohub_dapi_v1;
+  DROP VIEW biohub_dapi_v1.stakeholder_partnership;
+  DROP VIEW biohub_dapi_v1.project_first_nation;
+
+  SET search_path=biohub;
+
+  ----------------------------------------------------------------------------------------
+  -- Remove old 'stakeholder_partnership' and 'survey_first_nation_partnership' tables
+  ----------------------------------------------------------------------------------------
+
+  DROP TABLE stakeholder_partnership;
+  DROP TABLE project_first_nation;
```

Comment:
We need to migrate any existing prod data from the old tables to the new tables before we can drop the old tables.

Reply:
Totally forgot - done.